client/css/element-carousel.css

```diff
@@ -65,36 +56,56 @@
   padding: 1rem 2rem;
   background-color: rgba(0,0,0,0.35);
   white-space: normal;
+  color: white;
 }
 
-.element-carousel__title {
-  font-weight: bold;
+.element-carousel__content .element-carousel__title {
+  color: white;
```

Comment:
Colour stuff could be handled inside the theme?

Reply:
I agree. However, the overlay for the caption is quite dark, so by default, a body font colour might not be visible.
But happy for you to remove it.

Reply:
All good, forgot we had the caption background :)